src/main/java/school/lemon/changerequest/java/introduction/hw1/Task6.java

```diff
@@ -2,10 +2,13 @@
 
 public class Task6 {
     public static double calculateS(double x) {
-        return 0d;
-    }
+        return (1 + (x * (((Math.pow(x, 2) *x) *x) / (2 * (3*2) * (4*3*2)))));
```

Comment:
It's not correct solution.
Try to solve it step-by-step.
Also you can take look at other students solutions.